src/style/global.css

```
@@ -195,6 +196,34 @@
   font-weight: 700;
 }
 
+@utility headline1-extrabold {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)